test/fixtures/output/elixir/httpoison/application-json.ex

```diff
@@ -0,0 +1 @@
+HTTPoison.request(:post, "http://mockbin.com/har", ~s("{"number":1,"string":"f\"oo","arr":[1,2,3],"nested":{"a":"b"},"arr_mix":[1,"a",{"arr_mix_nested":{}}],"boolean":false}"), [{"content-type", "application/json"}])
```

Comment:
I don't have any Elixir experience but would/is it possible to break these arguments up into individual lines? Looks like Elixir has a [recommended 98 character line length](https://github.com/christopheradams/elixir_style_guide#formatting) and large request bodies will certainly blow past this.

Reply:
Switched to more concise style.